dnf-behave-tests/dnf/plugins-core/modulesync.feature

```diff
@@ -0,0 +1,57 @@
+@1868047
+Feature: Test for modulesync command
+
+
+Background:
+  Given I use repository "dnf-ci-fedora-modular"
+    And I use repository "dnf-ci-fedora"
+    And I enable plugin "modulesync"
+
+
+Scenario: I can download a module
+  Given I create directory "modulesync"
+    And I set working directory to "{context.dnf.installroot}/modulesync"
+    And I execute dnf with args "modulesync nodejs:8/minimal"
+   Then the exit code is 0
+    And stdout is
+    """
+    <REPOSYNC>
+    Directory walk started
+    Directory walk done - 5 packages
+    Loaded information about 0 packages
+    Temporary output repo path: {context.dnf.installroot}/modulesync/.repodata/
+    Preparing sqlite DBs
+    Pool started (with 5 workers)
+    Pool finished
+    """
+
+
+Scenario: I can download a module and init a modular repository with modulesync
+  Given I create directory "modulesync"
+    And I set working directory to "{context.dnf.installroot}/modulesync"
+    And I execute dnf with args "module download nodejs"
+   When I execute dnf with args "modulesync"
+   Then the exit code is 0
+    And stdout is
+    """
+    <REPOSYNC>
+    Directory walk started
+    Directory walk done - 0 packages
+    Temporary output repo path: {context.dnf.installroot}/modulesync/.repodata/
+    Preparing sqlite DBs
+    Pool started (with 5 workers)
+    Pool finished
+    """
```

Comment:
What about to test that a certain set of rpm was downloaded? I mean to test rpm files presence.